Kudu.Contracts/Functions/MasterKeyOperation.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace Kudu.Core.Functions
+{
+    public class MasterKeyOperation : IKeyOperation
+    {
+        public JObject GenerateKeyJson(string masterKey)
+        {
+            JObject hostJson = JObject.Parse($"{{\"masterKey\":{{\"name\":\"master\",\"value\":\"{masterKey}\",\"encrypted\": true }}}}");
```

Comment:
should we check for "functionkey" in version 0 as well for consistency? [ScriptSecretSerializerV0.CanSerialize()](https://github.com/fabiocav/azure-webjobs-sdk-script/blob/31cd7632375a65b290c08dd80b7a5a9fbdfb71e3/src/WebJobs.Script.WebHost/Security/ScriptSecretSerializerV0.cs#L81-L83)